content/reference/endpoints/reviews.md

```diff
@@ -3,44 +3,134 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents guests (travelers)' Reviews for the host for a given [Booking](/reference/endpoints/bookings/). 
```

Comment:
do we need the `'` after `(travelers)`?

Reply:
yeah, we do. I mean in singular you have `traveler's` and in plural `travelers'`, but it should be inside parentheses and applied to `guests'` as well :P